packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -42,7 +42,7 @@ export interface VpcNetworkProps {
     /**
      * The AWS resource tags to associate with the VPC.
      */
-    tags?: cdk.Tag[];
+    tags?: cdk.ITags;
```

Comment:
I think `string` should do since we will soon allow tokens to be represented as strings (#518). I don't think we really need to expose `propagate` in `props`. The 80% case (which we always try to optimize for) would just want to set key/value and `propagate: true`. We should obvsiouly document this behavior and explain how users can customize via `addTag`